--- /etc/dovecot/conf.d/10-mail.conf.orig       2023-10-10 07:57:12.813756297 +0000
+++ /etc/dovecot/conf.d/10-mail.conf    2023-10-10 08:07:13.601211003 +0000
@@ -28,6 +28,7 @@
 # <doc/wiki/MailLocation.txt>
 #
 #mail_location =
+mail_location = maildir:/var/mail/vhosts/%d/%n/Maildir

 # If you need to set multiple mailbox locations or want to change default
 # namespace settings, you can do it by defining namespace sections.
@@ -216,6 +217,7 @@
 # Space separated list of plugins to load for all services. Plugins specific to
 # IMAP, LDA, etc. are added to this list in their own .conf files.
 #mail_plugins =
+mail_plugins = quota

 ##
 ## Mailbox handling optimizations
