--- /etc/dovecot/conf.d/10-master.conf.orig     2023-10-10 07:58:48.431321199 +0000
+++ /etc/dovecot/conf.d/10-master.conf  2023-10-10 08:00:12.199833172 +0000
@@ -104,9 +104,11 @@
   }

   # Postfix smtp-auth
-  #unix_listener /var/spool/postfix/private/auth {
-  #  mode = 0666
-  #}
+  unix_listener /var/spool/postfix/private/auth {
+    mode = 0666
+    user = postfix
+    group = postfix
+  }

   # Auth process is run as this user.
   #user = $default_internal_user
