--- 10-ssl.conf.orig	2017-08-03 15:51:20.000000000 +0900
+++ 10-ssl.conf	2017-10-20 12:17:06.801631069 +0900
@@ -5,7 +5,8 @@
 # SSL/TLS support: yes, no, required. <doc/wiki/SSL.txt>
 # disable plain pop3 and imap, allowed are only pop3+TLS, pop3s, imap+TLS and imaps
 # plain imap and pop3 are still allowed for local connections
-ssl = required
+#ssl = required
+ssl = no
 
 # PEM encoded X.509 SSL/TLS certificate and private key. They're opened before
 # dropping root privileges, so keep the key file unreadable by anyone but
