--- /etc/postfix/master.cf.orig 2023-10-10 07:34:26.764111428 +0000
+++ /etc/postfix/master.cf      2023-10-10 07:35:02.299323101 +0000
@@ -14,7 +14,7 @@
 #smtpd     pass  -       -       n       -       -       smtpd
 #dnsblog   unix  -       -       n       -       0       dnsblog
 #tlsproxy  unix  -       -       n       -       0       tlsproxy
-#submission inet n       -       n       -       -       smtpd
+submission inet n       -       n       -       -       smtpd
 #  -o syslog_name=postfix/submission
 #  -o smtpd_tls_security_level=encrypt
 #  -o smtpd_sasl_auth_enable=yes
